config/action.d/pf.conf

```diff
@@ -75,14 +75,13 @@ protocol = tcp
 # Option:  port
 # Notes.:  specifies port to monitor
 # Values:  [ NUM | STRING ]  Default:
-#
-#port = telnet
+port = ssh
```

Comment:
Ahh, I see the port defined here will be not used at all, because uses always the port defined in jail.conf (or .local)